lib/ansible/modules/copy.py

```diff
@@ -63,11 +63,11 @@
   mode:
     description:
     - The permissions of the destination file or directory.
-    - For those used to C(/usr/bin/chmod) remember that modes are actually octal numbers.
-      You must either add a leading zero so that Ansible's YAML parser knows it is an octal number
-      (like C(0644) or C(01777)) or quote it (like C('644') or C('1777')) so Ansible receives a string
-      and can do its own conversion from string into number. Giving Ansible a number without following
-      one of these rules will end up with a decimal number which will have unexpected results.
+    - For those used to C(/usr/bin/chmod), remember that modes are actually octal numbers.
+      To prevent unexpected results of decimal to octal conversion, you B(must) add a leading zero
+      (like C(0644) or C(01777)) to the number. Though not required, you should also quote the number
+      (like C('0644') or C('01777')) to make it a string. The combination of these two elements guarantee
+      Ansible receives a string the YAML parser can interpret correctly into the desired permissions.
```

Comment:
```suggestion
      (like C(0644) or C(01777)) to the number. You can also wrap the number in single quotes
      (like C('0644') or C('01777')) to pass it to the YAML parser as a string. This guarantees that 
      Ansible correctly interprets the desired permissions.
```

Reply:
@thedoc31 Nice catch here and thanks for the contribution. Feel free to take or leave my suggestion.